### DISCUSSION TITLE
Why do only elements of MediumImportance widgets adhere to ThemedResource declarations with Fyne 2.5?

### DISCUSSION THREAD

**Discussion by Gambloide** [fyne-io/fyne#5020] [General] [answered]
If I go out of my way to declare resources as e.g. `SuccessThemedResource` then I expect them to adhere to the colors I defined in the theme for that type. However, with 2.5 this is no longer the case, and this expectation only holds for widgets with `MediumImportance`, though I have only tested it with buttons and their icons being `ThemedResources` specifically, since this is what broke for me.

I am not able to identify this as an intended change from the release notes. Can someone help me understand if this is now simply the new intended behavior, or if I need to change how to customize the theme or if it maybe something else entirely?

Simply changing which version of Fyne I import toggles this behavior without touching any other line of code.

**Fyne 2.4.3**
![image](https://github.com/user-attachments/assets/9215d2f6-0e97-45dc-872f-551fd77fe23d)
![image](https://github.com/user-attachments/assets/8f0fcc06-17ac-47a5-b79b-666eb3e3f6a4)

**Fyne v2.5**
![image](https://github.com/user-attachments/assets/d46a68bd-3d69-4089-8ffe-a06622914274)
![image](https://github.com/user-attachments/assets/728a2893-0302-4575-9cb3-1533f176beb7)

**Comment by andydotxyz** [comment, marked as answer]
I think this should be a bug report.
There was a change to ensure that text is more readable which could relate or it may relate to how icon theme changed - but what you report was not intended.

**Comment by Gambloide** [comment]
Got it. I'll try to make a minimal example to reproduce this then and create an issue! :+1:
